The syllabus (mostly the schedule) will be updated throughout the semester. You can see it as a <a href="https://github.com/bomeara/phylometh/raw/master/other/Syllabus.pdf">PDF</a> or <a href="https://github.com/bomeara/phylometh/blob/master/other/Syllabus.Rmd">RMarkdown</a> file; I have also embedded it below.
<div id="eeb603-phylogenetic-methods-phylometh.-spring-2016" class="section level2">
<h2>Phylogenetic Methods (EEB603: grad students; EEB504: advanced undergrads)</h2>
<h2>Syllabus version: 12:29 AM, 13 January, 2016</h2>
Instructor: <strong><a href="http://www.brianomeara.info">Brian O’Meara</a></strong> (<a href="mailto:bomeara@utk.edu">bomeara@utk.edu</a>)

Course website: <strong><a class="uri" href="http://phylometh.org">http://phylometh.org</a></strong>

Office Hours (446 Hesler) by appointment.
<div id="anonymous-feedback" class="section level4">
<h4>Anonymous feedback</h4>
<a class="uri" href="http://www.brianomeara.info/feedback">http://www.brianomeara.info/feedback</a>. This is the first time the course is being run, and it’s taught in a very different way from most courses. There is a lot that will go wrong. Let me know what’s not working while it can still improve.

</div>
<div id="what-you-will-learn" class="section level4">
<h4>What you will learn</h4>
This is a modern class in phylogenetic methods. You will learn how to get data, build a tree, use comparative methods, and how to make new methods. More importantly, you will learn <em>why</em> to do all this.

</div>
<div id="requirements" class="section level4">
<h4>Requirements</h4>
<ul>
 	<li>A laptop on which you can install software (have root / admin access)
<ul>
 	<li>It’ll be easiest if this is a Mac; less easy is Linux; Windows will be hard</li>
</ul>
</li>
 	<li>A question and some data to answer it</li>
 	<li>GitHub account</li>
 	<li>Gitter</li>
 	<li>R</li>
 	<li>And other software we’ll learn about during the course</li>
</ul>
</div>
<div id="evaluation" class="section level4">
<h4>Evaluation</h4>
Grading will be based on effort. Some of the things will be harder for some students than others just given differences in what they’ve been exposed to so far, and I don’t want to penalize students who are less skilled at R, for example. But I do require that you put in work on this class: work through the exercises, think about the papers, etc. Much of the homework is chewy, so it will take some thought to get through, and you might not be able to do all of this. That’s ok.

</div>
<div id="teaching" class="section level4">
<h4>Teaching</h4>
The class is flipped: rather than me droning at the front of the room, I’ll be releasing videos online for you to watch. They will be linked to from this syllabus. In class, talk about what was not clear from the videos about methods, and we’ll dig into that. We can also work on problematic parts of the exercises (you should do most of them before class) and talk about papers.

</div>
<div id="online-option" class="section level4">
<h4>Online option</h4>
There is an online option, as well. In this first running of the class, I’m not seeking to give out credit, but online visitors are encouraged to sign up.

</div>
Schedule
<div id="schedule" class="section level3">
<ul>
 	<li><a href="http://www.phylometh.org/2016/01/week-1-getting-started/">Week 1, Jan 14</a>
<ul>
 	<li>Topic: Getting ready</li>
 	<li>Items: R, github, gitter, unit testing</li>
 	<li>Instructions: <a class="uri" href="http://rpubs.com/bomeara/phylomethsoftwareinstall">http://rpubs.com/bomeara/phylomethsoftwareinstall</a></li>
 	<li>Exercise: <del><a href="https://classroom.github.com/assignment-invitations/c28cc1d5a6c9e8151508df973ab23e70">Getting Started</a></del> This assignment used GitHub Classroom; we've decided to just use regular GitHub (about of quarter of students got intermittent errors). So fork this: <a href="https://github.com/PhyloMeth/GettingStarted">https://github.com/PhyloMeth/GettingStarted</a>, then commit and push, then do a pull request.</li>
</ul>
</li>
 	<li><a href="http://www.phylometh.org/2016/01/week-2-getting-trees/">Week 2, Jan 21</a>
<ul>
 	<li>Topic: Gathering data</li>
 	<li>Items: Sequences, trees, name resolution, alignment</li>
 	<li>Exercise: <del><a href="https://classroom.github.com/assignment-invitations/68c7e2e5d8119a862b32417151ac505d">Get trees</a></del> This assignment used GitHub Classroom; we've decided to just use regular GitHub (about of quarter of students got intermittent errors). So fork this: <a href="https://github.com/PhyloMeth/GetTrees">https://github.com/PhyloMeth/GetTrees</a>, then commit and push, then do a pull request.</li>
</ul>
</li>
 	<li><a href="http://www.phylometh.org/2016/01/week-3-likelihood-parsimony-and-bayesian-approaches/">Week 3, Jan 28</a>
<ul>
 	<li>Topic: Building trees</li>
 	<li>Items: Likelihood; Bayes; Parsimony</li>
 	<li>Exercise and instructions are both <a href="http://www.phylometh.org/2016/01/week-3-likelihood-parsimony-and-bayesian-approaches/">here</a>; the reading is <a href="https://www.mendeley.com/groups/8111971/phylometh/papers/added/0/tag/week3/">here</a>.</li>
</ul>
</li>
 	<li><a href="http://www.phylometh.org/2016/02/week-4-gene-tree-species-tree-and-networks/">Week 4, Feb 4</a>
<ul>
 	<li>Topic: Building trees</li>
 	<li>Items: Gene tree species tree; Network</li>
</ul>
</li>
 	<li><a href="http://www.phylometh.org/2016/02/week-5-dating/">Week 5, Feb 11</a>
<ul>
 	<li>Topic: Dating trees</li>
 	<li>Items: Beast, r8s</li>
</ul>
</li>
 	<li><a href="http://www.phylometh.org/2016/02/week-6-correlations/">Week 6, Feb 18</a>
<ul>
 	<li>Topic: Correlating characters</li>
 	<li>Items: independent contrasts, Pagel 1994, Hansen OU</li>
</ul>
</li>
 	<li><a href="http://www.phylometh.org/2016/02/week-7-feb-25/">Week 7, Feb 25</a>
<ul>
 	<li>Topic: Continuous character models</li>
 	<li>Items: Surface, OUwie, geiger</li>
</ul>
</li>
 	<li>Week 8, Mar 3
<ul>
 	<li>Catch up week: my sense is that folks are having issues keeping up with the material, we're behind on videos, etc. So we will use this week to make sure everyone is caught up before moving forward.</li>
</ul>
</li>
 	<li><a href="http://www.phylometh.org/2016/03/week-9-discrete-traits/">Week 9, Mar 10</a>
<ul>
 	<li>Topic: Discrete character models</li>
 	<li>Items: corHMM, geiger</li>
</ul>
</li>
 	<li>Week 10, Mar 24
<ul>
 	<li>Topic: Catch up week 2</li>
</ul>
</li>
 	<li><a href="http://www.phylometh.org/2016/03/week-11-diversification-only/">Week 11, Mar 31</a>
<ul>
 	<li>Topic: Diversification models</li>
 	<li>Items: Sister group comparisons, BAMM</li>
</ul>
</li>
 	<li><a href="http://www.phylometh.org/2016/04/week-12-sse-models/">Week 12, Apr 7</a>
<ul>
 	<li>Topic: SSE models</li>
 	<li>Items: diversitree, hisse</li>
</ul>
</li>
 	<li><a href="http://www.phylometh.org/2016/04/week-13-14-build-a-method/">Week 13&amp; 14, Apr 14 &amp; April 21</a>
<ul>
 	<li>Topic: Building a new comparative method</li>
</ul>
</li>
 	<li>Week 11, April 28
<ul>
 	<li>Topic: Testing a new comparative method</li>
</ul>
</li>
</ul>
<strong>Course made possible by</strong>: NSF CAREER grant to O’Meara &amp; ongoing support from the Department of Ecology &amp; Evolutionary Biology, U. of Tennessee, Knoxville.

</div>
</div>
