---
title: 'PhyloMeth: Dating'
author: "Brian O'Meara"
output: html_document
---

*Still in progress*

##Objectives
By the end of this week, you will:

* Understand dating algorithms
* Be able to use r8s and BEAST
* Be afraid of calibrations

Make sure to **read the relevant papers**: https://www.mendeley.com/groups/8111971/phylometh/papers/added/0/tag/week5/    

To do this week's assignments, you will have to:

* **Download and install r8s** from http://loco.biosci.arizona.edu/r8s/ 
* **Download BEAST and Beauti** from http://beast.bio.ed.ac.uk
* **Install a tweaked version of Geiger**
    - `library(devtools)`
    - `install_github("bomeara/geiger-v2")` (eventually I'll make a pull request)
 
## BEAST 
For the BEAST part, we're not going to do our testing via R package approach: dealing with file paths and such are too problematic. So we'll just go through an exercise, but unlike many canned tutorials, you'll have to figure out what to do at stages. Note that this is based on the [tutorial](https://github.com/CompEvol/beast2/blob/master/doc/tutorials/DivergenceDating/DivergenceDatingTutorialv2.0.3.pdf?raw=true) by Drummond, Rambaut, and Bouckaert.

BEAST uses XML files for commands, rather than NEXUS files (though it does use NEXUS files for data). This XML format is different from [NeXML](http://www.nexml.org) and [PhyloXML](http://www.phyloxml.org), two other XML formats proposed for phylogenetics (though, as of now, they're still not used much in the field -- most students won't encounter them). These files are often made using the program BEAUTi (also from the BEAST developers) and then, sometimes, hand edited. 

1. **Import primates-mtDNA.nex** into BEAUTi; it's includided in examples/nexus in the BEAST folder. Note it's File -> Import alignment.
2. But wait -- what are you loading? Have you looked at the file? Open it in a text editor (or R, or Mesquite) and look at it. Do you believe the sequences are aligned properly? Is there anything weird about them? This is an essential step.
3. We will be partitioning, as with RAxML. This file already has some partitions, but some overlap (coding vs the three codon positions). Delete the coding partition using the `-` sign button on the bottom of the window.

**more here**
